config/config-api/src/main/java/com/thoughtworks/go/config/remote/ConfigRepoConfig.java

```diff
@@ -65,7 +67,6 @@ public void validate(ValidationContext validationContext) {
         this.validateRepoIsSet();
         this.validateMaterial(validationContext);
         if (isValidMaterial()) {
-            this.validateAutoUpdateEnabled();
             this.validateAutoUpdateState(validationContext);
```

Comment:
The implementation for `validateAutoUpdateState` has to change to check if `auto_update` value for ConfigRepo material matches with regular material.

Reply:
Is this because if there is an existing material that has the same config, `auto_update` must be the same?

Reply:
Changed this to test if autoUpdate is the same across all matching materials

Reply:
> Is this because if there is an existing material that has the same config, auto_update must be the same?
Yes